lib/Jira.js

```diff
@@ -84,8 +84,7 @@ class Jira {
                     repoName,
                     changeBranch,
                     branchName,
-                    jiraTargetEnv,
-                    reviewer
+                    JIRA_ENV_REVIEWER[i].REVIEWER
```

Comment:
Seems to have accidentally replaced the environment and the reviewer with just the reviewer.

Reply:
corrected